src/app/beer_garden/router.py

```diff
@@ -556,20 +562,18 @@ def _determine_target_garden(operation: Operation) -> str:
 
 def _forward_stomp(operation: Operation, target_garden: Garden):
     try:
-        if not stomp_garden_connections[target_garden.name].is_connected():
-            if not stomp_garden_connections[target_garden.name].connect():
-                raise ConnectFailedException()
+        conn = stomp_garden_connections[target_garden.name]
 
-        message, response_headers = process(operation)
+        if not conn.is_connected() and not conn.connect():
+            raise ConnectFailedException()
 
-        response_headers = consolidate_headers(
-            response_headers, target_garden.connection_params.get("stomp_headers")
-        )
+        header_list = target_garden.connection_params["stomp"].get("headers", {})
+        conn_headers = parse_header_list(header_list)
```

Comment:
Was the issue that we were not parsing the headers? Previously the headers were grabbed from the connection parameters. I am curious what was it trying to jam into the headers field since it didn't know what the Key/Value properties were.

Reply:
I'm not sure, but I think this problem was probably caused by #962, where I missed this part when switching to the nested connection parameters.

Before this PR headers taken from the config were strings, so they needed to be parsed. Headers taken from the database (aka, this section of code) weren't strings, so they didn't need to be parsed the same way. I'm not sure if they were being set correctly, though.